Techome/Techome/Views/TotalList/TotalListView.swift

```diff
@@ -0,0 +1,260 @@
+        static let dayRecordPadding: CGFloat = 4
+        static let caffeineRecordRowVerticalPadding: CGFloat = 15
+        static let caffeineRecordRowHorizontalPadding: CGFloat = 20
+        static let textVerticalPadding: CGFloat = 5
+        static let sideEffectRecordRowVerticalPadding: CGFloat = 15
+        static let sideEffectRecordRowHorizontalPadding: CGFloat = 20
+        static let sideEffectRecordCellHorizontalPadding: CGFloat = 21
+        static let sideEffectRecordCellVerticalPadding: CGFloat = 20
+        static let caffeineRecordAmountUnitPadding: CGFloat = 1
+        static let fullViewHorizontalPadding: CGFloat = 15
+        static let fullViewVerticalPadding: CGFloat = 23
+        static let dateVerticalPadding: CGFloat = 6
+        static let dividerHorizontalPadding: CGFloat = 15
+        
+    }
+    
+    struct Sizes {
+        static let mainWidth: CGFloat = UIScreen.main.bounds.width
+        static let mainHeight: CGFloat = UIScreen.main.bounds.height
+        static let cardWidth: CGFloat = UIScreen.main.bounds.width - 30
+        static let sideEffectRecordCellFixedWidth: CGFloat = 46
+    }
+    
+    
+}
+
+//카페인 + 부작용 리스트 병합에 사용
+struct tmpDataTotalList : Hashable {
+    var dataType: String
+    var dataIdx: Int
+}
+
+//카페인 + 부작용 리스트 병합 결과
+//(추후 구현) datatype과 idx 활용해 리스트에 넣을 알맞은 데이터 찾기
+private var tmpDataTotalListArr : [tmpDataTotalList] = [
+    tmpDataTotalList(dataType: "drink", dataIdx: 1),
+    tmpDataTotalList(dataType: "sideEffect", dataIdx: 1),
+    tmpDataTotalList(dataType: "drink", dataIdx: 2)
+]
+
+// 전체 리스트
+struct TotalListView: View {
+    
+    @Environment(\.presentationMode) var presentationMode
+    
+    var body: some View {
+        
+        ScrollView(){
+        
+            LazyVStack(spacing: 23){
+                ForEach(1..<10) { _ in
+                    TotalRecordsByDay(curDate: "2022.06.03")
+                }
+                
+                
+            }
+            .padding(.horizontal, TotalListLayoutValue.Paddings.fullViewHorizontalPadding)
+            .padding(.top, TotalListLayoutValue.Paddings.fullViewVerticalPadding) //navigation bar와 간격
+        }
+        .background(Color.backgroundCream)
+        .navigationTitle(Text("전체 리스트").font(.caption))
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden(true)
+        .toolbar{
+            ToolbarItem(placement: .navigationBarLeading){
+                        Button(action: {
+                            presentationMode.wrappedValue.dismiss()
+                        }) {
+                            Image(systemName: "chevron.left")
+                                .font(.headline)
+                                .foregroundColor(.primaryBrown)
+                        }
+                }
+        }
+    }
+}
+
+//날짜별 카페인 + 부작용 데이터 : 날짜별로 스트레스 + 부작용 데이터 받아와서 뷰 만들기
+//(추후) 카페인 데이터인지 부작용 데이터인지에 따라 맞는 리스트 보여주도록 구현
+struct TotalRecordsByDay: View {
+    let curDate : String
```

Comment:
```suggestion
    private let curDate : String
```

struct 내에서만 사용되는게 맞다면, 접근제한자 사용을 추천드려요 ~

Reply:
네 알겠습니다, 다른 부분에도 접근제한자 추가하겠습니다!